pycalphad/model.py

```diff
@@ -280,6 +295,8 @@ def symbol_replace(obj, symbols):
             # of other symbols
             for iteration in range(_MAX_PARAM_NESTING):
                 obj = obj.xreplace(symbols)
+                if iteration < 3:
```

Comment:
Why `3` here?

Reply:
This was some code that was originally operating on the entire Model AST, and then moved to be inside `symbol_replace` (see https://github.com/pycalphad/pycalphad/pull/431#issuecomment-1279630113 ). The iteration count limit was an attempt to work around an apparent bug, and also a (premature) performance optimization. It's no longer relevant in the current code and has been removed, so unwrapping will occur until the loop terminates. (In practice, there are rarely more than 3 levels deep of symbol nesting anyway.)